plugin/src/main/java/com/denizenscript/denizen/events/entity/EntityEntersBlockScriptEvent.java

```diff
@@ -0,0 +1,90 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.MaterialTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityEnterBlockEvent;
+
+public class EntityEntersBlockScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // <entity> enters <block>
+    //
+    // @Group Entity
+    //
+    // @Location true
+    //
+    // @Cancellable true
+    //
+    // @Triggers when an entity enters, and is stored in a block. Does not fire when a silverfish "enters" a stone block. Prefer <@link event <entity> changes <block> (into <block>)> for that.

```

Comment:
Considering this only fires for when bees enter beehives right now, I think you should include "For exmaple, this event is called for bees entering a bee hive." Right now all you have as an example is what the event *doesn't* do